presto-native-execution/presto_cpp/main/common/Counters.h

```diff
@@ -36,6 +36,18 @@ constexpr folly::StringPiece kCounterNumHTTPRequestError{
 constexpr folly::StringPiece kCounterHTTPRequestLatencyMs{
     "presto_cpp.http_request_latency_ms"};
 
+// Number of http client onBody calls in PrestoExchangeSource.
+constexpr folly::StringPiece kCounterHttpClientPrestoExchangeNumOnBody{
+    "presto_cpp.http.client.presto_exchange_source.num_onBody"};
```

Comment:
Leave an empty line here?

Reply:
Actually kCounterHttpClientPrestoExchangeNumOnBody and kCounterHttpClientPrestoExchangeOnBodyBytes are the same category, so I will leave them together